notascript/src/App.js

```diff
@@ -30,32 +32,40 @@ export default class App extends Component {
 				description:
 					'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec qu',
 				domains: ['Schule', 'Allgemein', 'Freizeit'],
-				symbols: ['ȁ', 'Ȃ', 'ȃ', 'Ȅ', 'ȅ', 'Ȇ', 'ȇ', 'Ȉ', 'ȉ', 'Ȑ'],
-				domains: ['tag1', 'tag2', 'tag3']
+				symbols: ['ȁ', 'Ȃ', 'ȃ', 'Ȅ', 'ȅ', 'Ȇ', 'ȇ', 'Ȉ', 'ȉ', 'Ȑ']
 			},
 			{
 				title: 'Dolor',
 				description:
 					'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec qu',
-				domains: ['Zeitung'],
+				domains: ['Zeitung', 'Medien'],
 				symbols: ['ȁ', 'Ȃ', 'ȃ', 'Ȅ', 'ȅ', 'Ȇ', 'ȇ', 'Ȉ', 'ȉ', 'Ȑ']
 			}
 		]
 	};
 
-	createNewDocument(data, history) {
+	getIndexByTitle(document) {
+		return this.state.documents.findIndex(
+			index => index.title === document.title
+		);
+	}
+
+	addDocument(data, history) {
+		console.log(data);
 		const newDocument = {
 			title: data.title,
 			description: data.description,
-			symbols: ['*'],
-			domains: [' tag']
+			symbols: data.symbols || ['*'],
+			domains: data.domains || [' tag']
 		};
 
 		this.setState({
 			documents: [newDocument, ...this.state.documents]
 		});
 
-		history.replace('/overview');
```

Comment:
maybe you could move the history.replace to the Form.js to the function that handles the event listener. Why do you need the if statement?